### ISSUE TITLE
Problema de atualização do campo tpAcConv nos eventos s1200 e s2299 

### ISSUE THREAD

**Issue by marcelo225** [tst-labs/esocial#407]
@tiagoben , boa tarde!

Tivemos um problema com relação ao campo **tpAcConv**, pois na versão eSocial S-1.2 o tem agora o range de **A** até **J** conforme o layout abaixo:

![image](https://github.com/tst-labs/esocial/assets/4791314/85ea0180-8fad-41a2-a523-7d3ba1e2668b)

Um dos erros que dá ao enviar o payload no ETJ dá essa seguinte mensagem:
`tpAcConv': [facet 'enumeration'] The value 'J' is not an element of the set {'A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I'}."`

Atualmente, este XSD tem o range de **A** até o **I**

https://github.com/tst-labs/esocial/blob/master/src/esocial-esquemas/src/main/resources/xsd/tipos.xsd#L2467

Vcs ainda vão atualizar, pois precisamos dessa nova versão que já foi atualizada em produção do eSocial, conforme a [Nota Técnica v. S-1.2 01/2023 - rev. 03/04/2024](https://www.gov.br/esocial/pt-br/documentacao-tecnica/manuais/nota-tecnica-s-1-2-01-2023-rev.pdf), no item 3.4 

![image](https://github.com/tst-labs/esocial/assets/4791314/3b4a7e54-4e99-4924-b2d7-4aa3ee5cc153)

Qual é o procedimento? Basta atualizar os XSDs na pasta do projeto ou precisa ser feita algumas mudanças no core do projeto?

Obrigado!

**Comment by tiagoben** [comment]
Olá @marcelo225 

Nesse caso é preciso atualizar a versão do XSD e gerar novamente os arquivo. Iremos liberar uma nova versão assim que possível.

**Comment by marcelo225** [comment]
Se puder, precisavamos gerar folhas para o pessoal RGPS que trabalham aqui no TJDFT.
Agradecido, @tiagoben !

**Comment by marcelo225** [comment]
@tiagoben , blz?
Vi que no [README.md](https://github.com/tst-labs/esocial/blob/master/src/esocial-esquemas/README.md) tem um tutorial que explica como fazer, certo?
Eu tenho que rodar esse comando `mvn clean package -Pgenerate-resources` dentro da pasta **esocial-esquemas** onde tem o pom.xml ?

**Comment by tiagoben** [comment]
Olá @marcelo225 
Tem um comando bash em  src/esocial-esquemas/generate-java-from-xsd.sh que auxilia nessa criação

**Comment by marcelo225** [comment]
> Olá @marcelo225 Tem um comando bash em src/esocial-esquemas/generate-java-from-xsd.sh que auxilia nessa criação

Certo. Eu vi aqui, mas esse .sh já está declarado no pom.xml dos sub-projetos **esocial-esquemas** e **esocial-comunicacao**. Em tese, não precisaria rodar eles diretamente. Eu to tentando gerar tudo via maven mesmo, usando **mvn verify** dentro do diretório principal (src). Vou testar localmente aqui no Tribunal e logo lanço um PR, ok?

**Comment by marcelo225** [comment]
@tiagoben , eu não tenho permissão para criar PR no projeto. Poderia dar essa permissão?

`git push upstream atualizacao_xsds_tst_nt-02-2024-rev-29-02-2024-prod-22-04
remote: Permission to tst-labs/esocial.git denied to marcelo225.
fatal: unable to access 'https://github.com/tst-labs/esocial.git/': The requested URL returned error: 403`

**Comment by tiagoben** [comment]
Bom dia @marcelo225 
Na verdade a melhor forma de fazer é criar um fork e solicitar um pull request a partir dele

**Comment by marcelo225** [comment]
@tiagoben, segue o PR: https://github.com/tst-labs/esocial/pull/408
To providenciando a atualização do Esquemas XSD eSocial - Leiautes v. S-1.2 (até NT 04/2024) - produção em 01/08/2024
